content/Solutions/Optimizations/Networking.md

```diff
@@ -0,0 +1,339 @@
+This includes accommodating normal data transfer rates and potential bursts of high-speed data.
+The network should provide low latency and high throughput to ensure efficient data transmission.
+
+### Network Architecture
+
+The overall network architecture and design play a significant role in supporting 100G networking.
+This includes considerations such as deployment of appropriate routers, switches, and other networking equipment capable of handling high-speed data flows.
+
+All components, such as optical transducers, should be compatible.
+Consult your network switch manufacturer's compatibility matrix to confirm.
+
+### Link Aggregation
+
+The increased bandwidth capacity of a 100G network can result in enhanced performance.
+However, implementing Link Aggregation Control Protocol (LACP) on 100G switches, particularly with Multi-Chassis Link Aggregations (MLAGs) and additional layers of abstraction, can introduce added complexity.
+This has the potential to result in performance issues and requiring more meticulous configuration.
+
+Implementing SMB Multichannel in a 100G environment can contribute to more efficient utilization of the available resources and improved overall performance for file sharing and data access.
+
+The ability of SMB Multichannel to establish multiple connections in parallel aligns well with the capabilities of a high-speed network.
+This parallelism can result in efficient use of the available bandwidth and improved performance, especially when dealing with large data sets.
+Optimized for Large Workloads:
+
+In scenarios involving substantial data transfers, such as large file copies or data-intensive applications, SMB Multichannel in a 100G environment can help optimize connections and adapt to the high-speed networking capabilities, leading to improved responsiveness and reduced transfer times.
+While SMB Multichannel itself is not specific to a particular network speed, its benefits become more pronounced and impactful in environments with higher bandwidth capacities, such as those provided by 100G networks.
+
+### Record Size Tuning
+
+The increased bandwidth of 100G networks has the potential to expose performance bottlenecks that have limited impacts in 10G or even 40G environments.
+This means that optimization practices, such as record size tuning, can have a significant impact on network performance.
+
+You should ensure that the record size for each dataset is in alignment with its I/O workload.
+To adjust record size in TrueNAS SCALE or CORE, got to Advanced/Other Options on the **Add** or **Edit Dataset** screen.
+
+For a detailed discussion of record size tuning, see [Tuning Recordsize in OpenZFS](https://klarasystems.com/articles/tuning-recordsize-in-openzfs/) from Klara.
+
+<hr>
+
+## Performance Testing
+
+After completing initial network configuration, you should obtain performance benchmarks and verify the configuration suits your use case.
```

Comment:
The best types of benchmarks are those which most closely resemble your real world workload. Synthetic benchmarks may not give you a full understanding of the performance characteristics of your system